scripts/build.sh

```diff
@@ -29,6 +29,7 @@ if [ -d ./proto ]; then
 fi
 
 git clone https://github.com/sparkswap/relayer-proto.git ./proto
+rm -rf ./proto/.git
```

Comment:
why did we make this a submodule? and why in this pr?

Reply:
@treygriffith this change was made in a prior PR and git only gives you a warning for this change. I can make the change in another PR or we can leave it here